ToyBox/Classes/MonkeyPatchin/BagOfPatches/MiscWrath.cs

```diff
@@ -699,7 +700,15 @@ public static void LoadString_ModTagPatch(ref string __result, LocalizedString _
                 }
             }
         }
-
+        [HarmonyPatch(typeof(DisableDeviceRestrictionPart))]
+        public static class DisableDeviceRestrictionPartPatch {
+            [HarmonyPatch(nameof(DisableDeviceRestrictionPart.CheckRestriction)), HarmonyPostfix]
+            public static void CheckRestriction_Patch(DisableDeviceRestrictionPart __instance) {
+                if (settings.togglelockjam) {
+                    __instance.Jammed = false; // Sea recommended a transpiler, but this seems easier and cleaner?
```

Comment:
What's the original method?
If it's a "prevent" jamming, is there a reason for not bool prefixing?

Reply:
A bool prefix would break someone else's changes. Not that anyone is probably overriding this method. Don't seen the point in copy/pasting the whole method for a bool prefix. See screenshot from DNSpy

![2024-12-28 13 52 20 - dnSpy v6 5 1 (64-bit,  NET)](https://github.com/user-attachments/assets/d3bbc0ff-65aa-4cc5-bc14-83b3b6cfaf97)

Reply:
Eh, if it works I guess it's fine.

Reply:
Sea did recommend a transpiler, which might have been better, but this is a lot cleaner IMO (because I don't have to write IL)